src/main.cpp

```diff
@@ -243,12 +238,7 @@ int main(int argc, char **argv) {
   if (getuid() != 0) {
     errorOut("You must be root to read or set encryption options on a drive!");
   }
-  logFile.open(LOGFILE, std::ios::app);
-  if (!logFile.is_open()) {
-    std::cout << "Warning: Could not open '" << LOGFILE
-              << "' for key change auditing!\n";
-  }
-  chmod(LOGFILE, 0600);
+  openlog("stenc", 0, LOG_USER);
```

Comment:
Or maybe this way `openlog("stenc", LOG_CONS, LOG_USER);`?
LOG_CONS - syslog() writes a message to the system console if it cannot send a message to the Syslog. It's unlikely, but not impossible.

Reply:
Seems reasonable.